src/wiki/development/build-instructions.md

```diff
@@ -166,36 +166,56 @@ We recommend using a build workflow based on MSYS2, as it's the easiest way to g
 ### Dependencies
 
 - [MSYS2](https://www.msys2.org/) - Software Distribution and Building Platform for Windows
-  - Make sure to follow all instructions on the webpage.
```

Comment:
BTW, I removed this line because the webpage originally mentioned updating MSYS2 with `pacman -Syu`
The instructions now say it updates automatically.